src/core/middleware/handlers/api/create-mock.handler.ts

```diff
@@ -0,0 +1,76 @@
+import * as http from 'http';
+import * as path from 'path';
+
+import * as fs from 'fs-extra';
+import { inject, injectable } from 'inversify';
+
+import { Configuration } from '../../../configuration';
+import { Mock } from '../../../mock/mock';
+import { MockResponse } from '../../../mock/mock.response';
+import { State } from '../../../state/state';
+import { HttpHeaders, HttpMethods, HttpStatusCode } from '../../http';
+import { ApplicableHandler } from '../handler';
+
+/**  Handler for creating and saving mocks in the mocks directory. */
+@injectable()
+export class CreateMockHandler implements ApplicableHandler {
+    MOCK_EXTENTIONS = {
```

Comment:
The mock extensions are configured in the config.

So we have to think of how to match that to the pattern specified in the processor options.

https://github.com/ng-apimock/core/blob/master/src/core/processor/processing.options.ts#L4

Reply:
you are right, I will look into that next PR